java/src/apps/gui3/TabbedPreferences.java

```diff
@@ -142,7 +142,7 @@ public synchronized int init() {
         detailpanel.setBorder(BorderFactory.createEmptyBorder(6, 3, 6, 6));
 
         save = new JButton(
-                rb.getString("ButtonSave"),
+                ConfigBundle.getMessage("ButtonSave"),
```

Comment:
Thanks, now that you mention it, I'd even made a note last time I looked at this.